app/experimenter/experiments/forms.py

```diff
@@ -860,19 +864,41 @@ def required_reviews(self):
             self["review_bugzilla"],
             self["review_qa"],
             self["review_relman"],
-        )
+        ]
+        return self.add_conditional_reviews(default_required)
+
+    def add_conditional_reviews(self, required_reviews):
+        for review, risk in self._required_reviews_mapping.items():
+            if risk:
+                self._remove_optional_label(review)
+                self._change_help_text_to_risk_help(review)
+                required_reviews.append(self[review])
+        return required_reviews
+
+    def _remove_optional_label(self, review):
+        self[review].label = self[review].label.strip("(Optional)")
```

Comment:
Do these mutate `self` or the fields in a way that exceeds the lifetime of the form? From a quick glance, it seems like they would, but maybe I'm missing something about how Django forms work.